### ISSUE TITLE
Can't create a server without auto_deploy - API error

### ISSUE THREAD

**Issue by TheProKiller756** [Pterodactyl/Panel#799]
* Panel or Daemon: Panel 
* Version of Panel/Daemon: 0.6.4
* Server's OS: Centos 7
* Your Computer's OS & Browser: Windows 10 & Nightly

## Add Details Below:

Hi, I'm using the API to create servers and I'm doing this request:
```
{  
   "name":"3buGu6f1_2",
   "user_id":"31",
   "auto_deploy":false,
   "option_id":4,
   "service_id":"1",
   "memory":"1024",
   "swap":"0",
   "cpu":"0",
   "io":"1000",
   "disk":"1024",
   "pack_id":"0",
   "location_id":"1",
   "startup":"java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar {{SERVER_JARFILE}}",
   "description":"Minecraft server for user 31",
   "env_BUNGEE_VERSION":"latest",
   "env_SERVER_JARFILE":"bungeecord.jar",
   "node_id":"1",
   "allocation_id":"3"
}
```
Pointing to: https://mydomain.com/api/admin/servers?include=allocations
And I'm getting this error:
```
{
    "error": {
        "auto_deploy": ["The auto deploy must be accepted."]
    }
}
```
If I activate auto_deploy it doesn't use the specified node, it uses a "random" node selected with auto deploy. I tryied the request through php with curl and directly with curl.
Thanks.

**Comment by DaneEveritt** [comment]
Its an issue with validation, it will be fixed in 0.7

**Comment by DaneEveritt** [comment]
Closing this as it will be addressed as part of 0.7's new API anyways and this issue will become redundant.
